src/components/ContentSidebar/ActivitySidebar.js

```diff
@@ -428,6 +434,22 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, file.id);
     };
 
+    renderAddTaskButton = () => {
+        const { isDisabled = false } = this.props;
```

Comment:
Seems best to leave the default value to the `defaultProps` above to avoid duplication/confusion.